Sprint-3/3-mandatory-practice/implement/get-ordinal-number.js

```diff
@@ -1,5 +1,27 @@
 function getOrdinalNumber(num) {
- return "1st";
+    let lastDigit = num % 10;
+    let lastTwoDig = num % 100;
+    if (lastTwoDig >= 11 && lastTwoDig <= 13) {
+        return num + "th";}
+    if (lastDigit == 1 ) {
+        return num + "st";
+    }
+    if (lastDigit == 2) {
+        return num + "nd";
+    }
+    if (lastDigit == 3) {
+        return num + "rd";
+    }
+
+    return num + "th";
 }
```

Comment:
Your code is working, it's good but we can also enhance this logic by checking types of arguments as well for instance what if we pass getOrdinalNumber("22") strings. consider this as a challange